charts/cp-schema-registry/values.yaml

```diff
@@ -83,6 +83,39 @@ resources: {}
 ## Custom pod annotations
 podAnnotations:
   cluster-autoscaler.kubernetes.io/safe-to-evict: "true"  # Allow the autoscaler to downscale the cluster.
+  sidecar.istio.io/inject: "false"
+  ad.datadoghq.com/cp-schema-registry-server.logs: '[{"source":"schema-registry","service":"{{ .Release.Name }}"}]'
```

Comment:
I would suggest to set all datadog parameters separately and add an enable option in order to have similar tempalte as what we have on the other charts (please check maria or cassandra charts for examples)

Reply:
+1 with Aymeric: we defined a "method" to integrate datadog in our charts (a `datadog` section in values.yaml, a function in _helpers.yaml) so let's keep it for all.
Except if there is an issue, but let's talk about it in that case :)

Reply:
Thanks guys, I will do the same as in the other charts.

Reply:
Honestly, now that I've done it, I think it adds a bit of complexity with the helpers for not much, it's the same amount of text we need to add to the same values file, plus a boolean to disable the datadog integration which doesn't seem very useful.